src/util.js

```diff
@@ -3,9 +3,38 @@
 const {
   hasNewline,
   skipEverythingButNewLine,
-  skipNewline
+  skipNewline,
+  isNextLineEmpty: _isNextLineEmpty,
+  isPreviousLineEmpty: _isPreviousLineEmpty,
+  getNextNonSpaceNonCommentCharacterIndex: _getNextNonSpaceNonCommentCharacterIndex,
 } = require("prettier").util;
 
+const prettierVersion = require("prettier").version;
+
+function lookupIfPrettier2(options, prop) {
+  return parseInt(prettierVersion[0]) > 1 ? options[prop] : options;
+}
+
+function isPreviousLineEmpty(text, node, options) {
+  return _isPreviousLineEmpty(
+    text,
+    node,
+    lookupIfPrettier2(options, "locStart")
+  );
+}
+
+function isNextLineEmpty(text, node, options) {
+  return _isNextLineEmpty(text, node, lookupIfPrettier2(options, "locEnd"));
+}
+
+function getNextNonSpaceNonCommentCharacterIndex(text, node, options) {
+  return _getNextNonSpaceNonCommentCharacterIndex(
+    text,
+    node,
+    lookupIfPrettier2(options, "locEnd")
+  );
+}
```

Comment:
These are the actual code changes

(probably should've done a separate commit for the formatting changes :see_no_evil:)